extensions/sql-database-projects/src/models/project.ts

```diff
@@ -195,6 +187,7 @@ export class Project {
 		}
 		else {
 			this.projFileXmlDoc.documentElement.appendChild(importNode, oldImportNode);
+			this.importedTargets.push(project);	// Add new import target to the list
```

Comment:
Can `project` be renamed to something like `projectAttributeVal`?  This keeps throwing me off because it looks like we're adding a `Project`-type object to a list of targets.

Reply:
Sure, I can definitely fix that.